connection.js

```diff
@@ -100,6 +100,7 @@ class Connection {
         this.transaction = null;
         this.tran_count = 0;
         this.capabilities = null;
+        this.ehlo_hello_message = config.get('ehlo_hello_message') || 'Haraka is at your service.'
```

Comment:
The new default Haraka message is missing `, ` from the previous.
`Hello [127.0.0.1]Haraka is at your service.`

Reply:
What you want to do is check if ehlo_hello_message is set and if so, add ', ' to the message. Although in that case someone is forced to use that ', ' as it will be static. Other solution is to add ', ' to this to the default text.

Reply:
In this case it would probably be easiest with just add it to the default message?